src/index.js

```diff
@@ -23,6 +24,13 @@ export default function* () {
 
   } else {
     const path = yield resolve({ file, name: "Flyfile" })
+    const ext = path.split(".")[path.split(".").length - 1]
+    const modules = jsVariants["." + ext]
+    let extModule = modules
+    if(Array.isArray(modules)) {
+      extModule = modules[0]
+    }
+    require(extModule)
```

Comment:
Yes, if user want to `x` language but user does not install `x` language, `require` throws Error. :hamster: